third_party/2and3/mypy_extensions.pyi

```diff
@@ -1,8 +1,21 @@
-from typing import Dict, Type, TypeVar, Union
+from typing import Dict, Type, TypeVar, Optional, Union
 
 _T = TypeVar('_T')
 
```

Comment:
I am a smartass and put this as a separate diff because it is unrelated: https://github.com/python/typeshed/pull/1237